floater/test/test_utils.py

```diff
@@ -95,24 +95,37 @@ def test_floats_to_netcdf(tmpdir, mitgcm_float_datadir_csv):
 
     input_dir = str(mitgcm_float_datadir_csv) + '/'
     output_dir = str(tmpdir) + '/'
-
     os.chdir(input_dir)
+
+    # least options
+    utils.floats_to_netcdf(input_dir=input_dir, output_fname='test')
+    # most options
```

Comment:
@rabernat The first and the second calls output NetCDF files into different directories, i.e. `input_dir/test.nc` by default, and `output_dir/test.nc` specified in the most options test.